src/Language/Haskell/HGrep/Print.hs

```diff
@@ -74,14 +77,14 @@ printSearchResult (PrintOpts co lno) (SearchResult anns ast) =
     wholeSrc :: [Char]
     wholeSrc = EP.exactPrint ast anns
 
-    nill, src :: [[Char]]
-    (nill, src) = L.span null $ L.lines wholeSrc
+    src :: [[Char]]
+    (_, src) = L.span (\s -> null s || all isSpace s) $ L.lines wholeSrc
```

Comment:
If there is no more need in empty lines you can use [`L.dropWhile`](http://hackage.haskell.org/package/base-4.10.0.0/docs/Prelude.html#v:dropWhile) instead of `L.span`

Reply:
Not sure about exact print and haven't wrapped my head around all of the new additions but is it possible there would be empty lines after the definitions? Perhaps I should filter instead of dropWhile

Reply:
Sorry, exact-print*

Reply:
We can also just use `all isSpace` section here since it's vacuously true on empty strings / lists.